### PR TITLE
Add image quality setting for image sizes

### LINKED ISSUE #5588: Configure jpeg_quality per image size
### Description

It should be possible to set the `jpeg_quality` per image size because for small images it makes sense to set it higher than for larger ones.

See discussion https://contao.slack.com/archives/CK4J0KNDB/p1671191941333959
@ausi

### PR DESCRIPTION
**Pull request by ausi**
Implements #5588

### CODE REVIEW

**Review by Toflar — CHANGES_REQUESTED** [review]
It's exactly like we've discussed 🥳 ❤️ 

I think the label for the field in `tl_image_size` is missing :)

**Comment by ausi** [comment]
> I think the label for the field in `tl_image_size` is missing :)

Done in 8e62f4fe3637a95ab148865f5829976623008ee2

File: core-bundle/contao/languages/en/tl_image_size.xlf [hunk 1]
```diff
@@ -77,6 +77,12 @@
       <trans-unit id="tl_image_size.skipIfDimensionsMatch.1">
         <source>Note that no optimizations such as metadata stripping or JPEG compression will be applied in this case.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.imageQuality.0">
+        <source>Image quality</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.imageQuality.1">
+        <source>Image compression quality between 0 (lowest quality) and 100 (lossless).</source>
```
Comment (Toflar) [inline core-bundle/contao/languages/en/tl_image_size.xlf]:
```suggestion
        <source>Optional image compression quality between 0 (lowest quality) and 100 (lossless). If left empty, the system-wide configuration applies.</source>
```

wdyt?

**Review by Toflar — DISMISSED** [review]
Another great neat feature for the Core 🥳

File: core-bundle/contao/languages/en/tl_image_size.xlf [hunk 2]
```diff
@@ -77,6 +77,12 @@
       <trans-unit id="tl_image_size.skipIfDimensionsMatch.1">
         <source>Note that no optimizations such as metadata stripping or JPEG compression will be applied in this case.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.imageQuality.0">
+        <source>Image quality</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.imageQuality.1">
+        <source>Optional image compression quality between 0 (lowest quality) and 100 (lossless). If left empty, the system-wide configuration applies.</source>
```
Comment (fritzmg) [inline core-bundle/contao/languages/en/tl_image_size.xlf]: Minor nitpick: with WebP and JPEG the compression is never lossless, even at 100% quality, is it?

Comment (Toflar) [reply core-bundle/contao/languages/en/tl_image_size.xlf]: https://imagemagick.org/script/webp.php

Comment (fritzmg) [reply core-bundle/contao/languages/en/tl_image_size.xlf]: Yes, it's only lossless if enabled (which in our case it is, admittedly). Anyway my suggestion would be to just write "highest quality".

Comment (leofeyer) [reply core-bundle/contao/languages/en/tl_image_size.xlf]: I have adjusted this in a1ff03f9c1af080c94b1e15589f0940ad2b189f1.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**